app/routes/events/components/EventAdministrate/RegistrationTables.tsx

```diff
@@ -190,32 +236,30 @@ export class RegisteredTable extends Component<Props> {
         title: 'Bruker',
         dataIndex: 'user',
         search: true,
+        notCentered: true,
```

Comment:
I know this isn't you but this is just stupid 🙃

Reply:
Kinda stupid, but also very useful. Not all columns should be centered.

Reply:
The point was just that `notCentered=false === centered=true`

Reply:
Ye, but the default should be centered, so this just makes for less config.

Reply:
Ah i thought this was here from before. But I see now that you changed it to the "not" version. Then I can't let it pass. Having any boolean prop/arg beginning with "not" is a huge nono. Please change to centered and adjust the default accordingly.

Reply:
What? You want the default to be not centered? Passing `centered: false` is dumb. But whatever, I'm tired of this.